conversion/conversion.go

```diff
@@ -64,10 +66,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	dydb "github.com/aws/aws-sdk-go/service/dynamodb"
 	"github.com/aws/aws-sdk-go/service/dynamodbstreams"
+	mysqldriver "github.com/go-sql-driver/mysql"
 	"go.uber.org/zap"
 	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/protobuf/proto"
+	"github.com/jackc/pgx/v5"
+    "github.com/jackc/pgx/v5/stdlib"
```

Comment:
formatting differences. Please run go mod tidy

Reply:
Not sure why it is showing like that here, on my IDE it looks fine. Ran `go mod tidy` once to be sure.